### ISSUE TITLE
A11y_ML .NET Model Builder_ Scenario_ Screen Reader: Narrator/NVDA is not reading name for "three dots(...)" control present at select a folder

### ISSUE THREAD

**Issue by InduPriya1805** [dotnet/machinelearning-modelbuilder#1135]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHubTags:
#A11yMAS; #A11yTCS; #Win32; #DesktopApp; #Feature-ML.NETModelBuilder-Win32-Nov2020; #A11ySev1; #Benchmark; #Visual Studio; #Screen Reader; #Narrator; #NVDA; #WCAG1.3.1

### Environment Details:
Microsoft Visual Studio

### Note:
This issue is also observed in "**Object Detection-Data Input**" scenario

### Repro Steps:

1. Turn on screen reader Narrator/NVDA
2. Open Visual Studio Enterprise 2019 Preview
3. Create a new .Net Core Console App (C#)
4. Tab to "Solution Explorer" and right click on the project (Shift + F10) and then from the expanded list Select "Add" and then from the list items select "Machine learning"
5. Verify whether "Feedback" is accessible using keyboard
6. Tab till 'Image Classification' and press enter
7. Environment tab will open which is having 'Local CPU environment' in selected state. Tab till 'Next Step' button and press enter
8. Tab till three dots  "..." button present at 'Select a folder' and verify how screen reader is reading

### Actual Result:
Narrator/NVDA is not reading name for "three dots(...)" control present at select a folder

### Expected Result:
Narrator/NVDA should read name for "three dots(...)" control present at select a folder as "Browser the folder"

### User Impact:
Visually impaired users will not know about the label name and its purpose if screen reader does not reading anything

### Attachment for Reference:
[Bug_1135_Narrator is not reading button name.pptx](https://github.com/dotnet/machinelearning-modelbuilder/files/5635476/Bug_1135_Narrator.is.not.reading.button.name.pptx)
[Bug_1135_NVDA is not reading name for button.pptx](https://github.com/dotnet/machinelearning-modelbuilder/files/5635512/Bug_1135_NVDA.is.not.reading.name.for.button.pptx)

**Comment by SaisreeDandamudi** [comment]
[Bug_1135_Fixed_Narrator.is.not.reading.button.name.pptx](https://github.com/dotnet/machinelearning-modelbuilder/files/6042401/Bug_1135_Fixed_Narrator.is.not.reading.button.name.pptx)
[Bug_1135_fixed_NVDA.is.not.reading.name.for.button.pptx](https://github.com/dotnet/machinelearning-modelbuilder/files/6042409/Bug_1135_fixed_NVDA.is.not.reading.name.for.button.pptx)

Verified the bug in the latest build and the issue is fixed
Hence closing the bug
Please find the attachments for reference
Microsoft Visual Studio Enterprise 2019 Preview
Version 16.9.0 Preview 4.0

**Comment by poojapatil36** [comment]
Github tag: #Closed;
